internal/backends/winit/glwindow.rs

```diff
@@ -26,7 +26,8 @@ use winit::dpi::LogicalSize;
 /// GraphicsWindow is an implementation of the [PlatformWindow][`crate::eventloop::PlatformWindow`] trait. This is
 /// typically instantiated by entry factory functions of the different graphics back ends.
 pub(crate) struct GLWindow<Renderer: WinitCompatibleRenderer + 'static> {
-    self_weak: Weak<corelib::window::WindowInner>,
+    window: corelib::api::Window,
+    self_weak: Weak<Self>,
```

Comment:
We could remove this in favor of `window.window_handle().platform_window().as_any().downcast_ref::<_>()` but I'm not sure it's worth it. This seems simpler, at the expense of memory.

Reply:
i think that extra pointer is fine :-)